lib/puppet/configurer.rb

```diff
@@ -246,9 +266,16 @@ def run_internal(options)
   private :run_internal
 
   def send_report(report)
+    report.finalize_report
+    report_send_time = thinmark {
+      Puppet::Transaction::Report.indirection.save(report, nil, :environment => Puppet::Node::Environment.remote(@environment)) if Puppet[:report]
+    }
+    # Includes the indirection save time in the last_run_summary.yaml
+    # and in the summary
+    report.add_times(:reporting, report_send_time)
+    report.finalize_report
     puts report.summary if Puppet[:summarize]
     save_last_run_summary(report)
-    Puppet::Transaction::Report.indirection.save(report, nil, :environment => Puppet::Node::Environment.remote(@environment)) if Puppet[:report]
```

Comment:
@joshcooper The only way I see is to log it instead of putting it in the report. But as @hlindberg said in the [ticket](https://tickets.puppetlabs.com/browse/PUP-4634), it adds bloat and overhead to the logs. 

Logging it is fine for me.